doc/admx/DesktopAppInstaller.admx

```diff
@@ -146,5 +146,15 @@
         <decimal value="0" />
       </disabledValue>
     </policy>
+    <policy name="EnableWindowsPackageManagerCommandLineInterfaces" class="Machine" displayName="$(string.EnableWindowsPackageManagerCommandLineInterfaces)" explainText="$(string.EnableWindowsPackageManagerCommandLineInterfacesExplanation)" key="Software\Policies\Microsoft\Windows\AppInstaller" valueName="EnableWindowsPackageManagerCommandLineInterfaces">

```

Comment:
Yeah, it would make sense to have them prefixed, but the others are not and that's hard to change. I can see the point of having a prefix, but I also think there is some value in having them all be consistent.

And if we are prefixing, does it need to be with WindowsPackageManager? Could it be just WinGet or something short?

Reply:
In this context, we shouldn't be prefixing our GPOs with an abbreviation or shortened form of our product name. Which is why we are pushing for the "Windows Package Manager" as the new prefix.